pkg/clustersync/clusterSync.go

```diff
@@ -69,10 +69,10 @@ func syncClusters(ctx context.Context) {
 			klog.V(4).Info("clusterWatch: UpdateFunc for", next.(*unstructured.Unstructured).GetKind())
 			processClusterUpsert(next)
 		},
-		// DeleteFunc: func(obj interface{}) {
-		// 	klog.Info("clusterWatch: DeleteFunc")
-		// 	processClusterDelete(obj)
-		// },
+		DeleteFunc: func(obj interface{}) {
+			klog.Info("clusterWatch: DeleteFunc")
```

Comment:
1. Add V(4)
2. Delete `clusterWatch: ` the logger automatically adds filename. Same with the previous 2 logs.
```suggestion
			klog.V(4).Info("DeleteFunc")
```